### ISSUE TITLE
Submission for RedHat compatibility

### ISSUE THREAD

**Issue by pulse-mind** [viasite-ansible/ansible-role-apache-vhosts#1]
Hello,

I'd like to add the RedHat (centOS7) compatibility. But I can not commit so these are the files, can you please add them ? 

tasks/configure-RedHat.yml:
```
---
- name: Copy welcome file
  copy: remote_src=True src={{ apache_conf_path }}/conf.d/welcome.conf dest={{ apache_conf_path }}/conf.d/welcome.conf.backup

- name: Remove welcome file
  file: path={{ apache_conf_path }}/conf.d/welcome.conf
  
- name: Create the directories sites-available and sites-enabled
  file:
    path="{{ apache_conf_path }}/{{ item }}"
    state=directory  
  with_items:
    - sites-enabled
    - sites-available  
  notify:
   - restart apache

- name: Add sites-enabled Include to httpd.conf
  lineinfile:
    dest={{ apache_conf_path }}/conf/httpd.conf
    state=present
    line='IncludeOptional sites-enabled/*.conf'
  notify:
    - restart apache

- name: Create the configurations for sites
  template:
    src: "{{ item.value.template | default(apache_vhosts_template) }}"
    dest: "{{ apache_conf_path }}/sites-available/{{ item.key }}.conf"
  with_dict: "{{ apache_vhosts_sites }}"
  when: item.key not in apache_vhosts_remove_sites
  notify:
   - restart apache

- name: Create links for sites-enabled
  file:
    state: link
    src: "{{ apache_conf_path }}/sites-available/{{ item.key }}.conf"
    dest: "{{ apache_conf_path }}/sites-enabled/{{ item.key }}.conf"
  with_dict: "{{ apache_vhosts_sites }}"
  when: item.key not in apache_vhosts_remove_sites
  notify:
   - restart apache

- name: Remove unwanted sites
  file:
    path: "{{ apache_conf_path }}/{{ item[0] }}/{{ item[1] }}.conf"
    state: absent
  with_nested:
    - [ 'sites-enabled', 'sites-available']
    - "{{ apache_vhosts_remove_sites }}"
  notify:
   - restart apache
```

vars/RedHat.yml:
```
---
apache_service: httpd
apache_daemon: httpd
apache_daemon_path: /usr/sbin/
apache_conf_path: /etc/httpd
```

**Comment by popstas** [comment]
Hello, I'm using Debian/Ubuntu only at this time, but if you want, you can send me pull request, I will merge it in next week. Also, maybe you found useful similar role for nginx - https://github.com/viasite-ansible/ansible-role-nginx-vhosts.

I made these roles for internal using, but I'm ready to support of other operating systems myself if it consume less than hour of my time and if I will receive some more requests.

**Comment by pulse-mind** [comment]
It is up to you, otherwise I will fork it.

Le jeu. 10 août 2017 18:32, Stanislav Popov <notifications@github.com> a
écrit :

**Comment by popstas** [comment]
Fixed in #2
